storybook/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {AppRegistry, Platform} from 'react-native';
+import {getStorybookUI, configure} from '@storybook/react-native';
+
+import {loadStories} from './storyLoader';
+
+import './rn-addons';
```

Comment:
do we need this? maybe add the imports from the other file here ?  is there a reason why you have them in separate files ?